mne/io/bti/bti.py

```diff
@@ -1170,8 +1170,11 @@ def _get_bti_info(pdf_fname, config_fname, head_shape_fname, rotation_x,
     chs = []
 
     # Note that 'name' and 'chan_label' are not the same.
-    # We want the configured label if out IO parsed it.
-    bti_ch_names = [c.get('chan_label', c['name']) for c in bti_info['chs']]
+    # We want the configured label if out IO parsed it
+    # except for the MEG channels for which we keep the config name
+    bti_ch_names = [c.get('chan_label' if c['yaxis_label'] != 'T' else 'name',
+                          c['name']) for c in bti_info['chs']]
```

Comment:
wondering how general this check is across systems. You are checking essentially the unit right? Keep in mind the 4D gradiometer system ... but I haven't seen any file from it so far, so fair enough.